### PR TITLE
Add support for Aqara Wireless Remote Switch H1 (Double Rocker) WRS-R02

### PR DESCRIPTION
**Pull request by SwoopX**
See #4842.

This PR also adds writing the multi-click attribute `0x0125` for Aqara Opple switches (lumi.remote.b686opcn01, lumi.remote.b486opcn01 and lumi.remote.b286opcn01).

### CODE REVIEW

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -1085,7 +1086,8 @@ void DeRestPluginPrivate::apsdeDataIndication(const deCONZ::ApsDataIndication &i
                              sensorNode->modelId().startsWith(QLatin1String("Lightify Switch Mini")) ||  // Osram 3 button remote
                              sensorNode->modelId().startsWith(QLatin1String("Switch 4x EU-LIGHTIFY")) || // Osram 4 button remote
                              sensorNode->modelId().startsWith(QLatin1String("Switch 4x-LIGHTIFY")) || // Osram 4 button remote
-                             sensorNode->modelId().startsWith(QLatin1String("Switch-LIGHTIFY")) ) // Osram 4 button remote
+                             sensorNode->modelId().startsWith(QLatin1String("Switch-LIGHTIFY")) || // Osram 4 button remote
+                             sensorNode->modelId() == QLatin1String("lumi.remote.b28ac1"))    // Aqara wireless remote switch H1 (double rocker)
```
Comment (manup) [inline de_web_plugin.cpp:1090]: This shouldn't compile?

File: de_web_plugin.cpp [hunk 2]
```diff
@@ -5955,7 +5958,8 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                              modelId.startsWith(QLatin1String("ZG2833PAC")) || // Sunricher C4
                              modelId.startsWith(QLatin1String("ED-1001")) ||  // EcoDim switches
                              modelId.startsWith(QLatin1String("45127")) ||    // Namron switches
-                             modelId.startsWith(QLatin1String("S57003")))     // SLC 4 ch remote switch
+                             modelId.startsWith(QLatin1String("S57003")) ||   // SLC 4 ch remote switch
+                             modelId == QLatin1String("lumi.remote.b28ac1"))  // Aqara Wireless Remote Switch H1 (Double Rocker)
```
Comment (manup) [inline de_web_plugin.cpp:5962]: This shouldn't compile?

File: de_web_plugin.cpp [hunk 3]
```diff
@@ -16357,16 +16362,25 @@ void DeRestPluginPrivate::delayedFastEnddeviceProbe(const deCONZ::NodeEvent *eve
...
+                deCONZ::ZclAttribute attr(0x0009, deCONZ::Zcl8BitUint, QLatin1String("mode"), deCONZ::ZclReadWrite, false);
                 attr.setValue(static_cast<quint64>(1));
-                writeAttribute(sensor, sensor->fingerPrint().endpoint, XIAOMI_CLUSTER_ID, attr, VENDOR_XIAOMI);
+                writeAttribute(sensor, 0x01, XIAOMI_CLUSTER_ID, attr, VENDOR_XIAOMI);
+
+                // Activate multiclick mode
+                DBG_Printf(DBG_INFO, "Write Aqara switch 0x%016llX multiclick mode attribute 0x0125 = 2\n", sensor->address().ext());
+                deCONZ::ZclAttribute attr2(0x0125, deCONZ::Zcl8BitUint, QLatin1String("multiclick mode"), deCONZ::ZclReadWrite, false);
+                attr2.setValue(static_cast<quint64>(2));
+                writeAttribute(sensor, 0x01, XIAOMI_CLUSTER_ID, attr2, VENDOR_XIAOMI);
```
Comment (manup) [inline de_web_plugin.cpp:16382]: Is it safe to assume hard coded endpoint 0x01 here for "86opcn01" and "b28ac1", or is the formerly used `sensor->fingerPrint().endpoint` a better fit?

**Review by manup — COMMENTED** [review]
Some smaller questions in the comments.

The PR description should also mention that the multi-click attribute `0x0125` is configured for "86opcn01" (human readable switch name? ;) as well.

File: de_web_plugin.cpp (hunk 1 above)
Comment (SwoopX) [reply de_web_plugin.cpp:1090]: Compiles fine?

File: de_web_plugin.cpp (hunk 2 above)
Comment (SwoopX) [reply de_web_plugin.cpp:5962]: Compiles fine?

File: de_web_plugin.cpp (hunk 3 above)
Comment (SwoopX) [reply de_web_plugin.cpp:16382]: Yes, endpoint 0x01 should be save. I guess I can amend that.

File: de_web_plugin.cpp (hunk 1 above)
Comment (manup) [reply de_web_plugin.cpp:1090]: Ahh sorry my bad, I expected `)))` at the end due the former version having `startsWidth(QLatin1String(...)))` at the *if* condition :smile:

**Pull request merged**